src/main/groovy/org/mockito/release/internal/comparison/PomFilter.groovy

```diff
@@ -0,0 +1,47 @@
+package org.mockito.release.internal.comparison
+
+class PomFilter {
```

Comment:
This class is fantastic! I really like the API and the model, and neat unit test. Thanks!